GUIDE.md

```diff
@@ -160,4 +160,90 @@ nvm use 20
 ### Install snarkjs
 ```sh
 npm install -g snarkjs
-```
\ No newline at end of file
+```
+
+## Two phases after es-node launch
+
+After the launch of ES node, it basically goes through two main stages.
+
+### Data sync phase
+
+The es-node will synchronize data from other peers in the network. You can check from the console the number of peers the node is connected to and, more importantly, the estimated syncing time.
+
+During this phase, the CPUs are expected to be fully occupied for data processing. If not, please refer to [the FAQ](#how-to-tune-the-performance-of-syncing) for performance tuning on this area.
+
+A typical log entry in this phase appears as follows:
+
+```
+INFO [01-18|09:13:32.564] Storage sync in progress progress=85.50% peerCount=2 syncTasksRemain=1@0 blobsSynced=1@128.00KiB blobsToSync=0 fillTasksRemain=30 emptyFilled=3,586,176@437.77GiB emptyToFill=608,127   timeUsed=1h48m7.556s  eta=18m20.127s
+
+```
+### Sampling phase 
+
+Once data synchronization is complete, the es-node will enter the sampling phase, also known as mining.
+
+A typical log entry of sampling during a slot looks like this:
+
+```
+INFO [01-19|05:02:23.210] Mining info retrieved                    shard=0 LastMineTime=1,705,634,628 Difficulty=4,718,592,000 proofsSubmitted=6
+INFO [01-19|05:02:23.210] Mining tasks assigned                    shard=0 threads=64 block=10,397,712 nonces=1,048,576
+INFO [01-19|05:02:26.050] The nonces are exhausted in this slot, waiting for the next block samplingTime=2.8s shard=0 block=10,397,712
+
+```
+When you see "The nonces are exhausted in this slot...", it indicates that your node has successfully completed all the sampling tasks within a slot. 
+The "samplingTime" value informs you of the duration, in seconds, it took to complete the sampling process.
+
+If the es-node doesn't have enough time to complete sampling within a slot, the log will display "Mining tasks timed out". For further actions, please refer to [the FAQ](#what-can-i-do-about-mining-tasks-timed-out).
+
+## FAQ
+
+### What should I do when the log frequently shows "i/o deadline reached" during data syncing?
+
+The timeout issue could be due to network reasons. It may be because the amount of data requested relative to network performance is too large, so even a slight delay in individual requests will be magnified.
+
+You can change the size of each request to a smaller value using `--p2p.max.request.size`. 
+The default value is `4194304`. You can try adjusting it to `1048576`.
+
+### How to tune the performance of syncing? 
```

Comment:
One related FAQ here: Why is the CPU fully utilized when syncing data? Does this imply that running an es-node requires high-end hardware? Additionally, is there a way to configure it to use less CPU power? I'm asking because I need to run other software on the same machine.